### PR TITLE
fix rte scroll handler firing on child scroll

### PR DESCRIPTION
**Pull request by erquhart**
**- Summary**

Code blocks can scroll horizontally in the rich text editor, but those scrolls were
propagating and triggering the scroll handler for the StickyContext component,
causing the toolbar to stick to the top of the scroll context.

This fix ensures that only context scrolls trigger the handler.

Also, fixed an issue where the recently added in-publish script fails CI.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,7 +9,6 @@
     "test:watch": "jest --watch",
     "build": "NODE_ENV=production webpack --config webpack.prod.js",
     "build:scripts": "NODE_ENV=production webpack --config webpack.cli.js",
-    "prepublish": "in-publish && npm run build || not-in-publish",
```
Comment (Benaiah) [inline package.json]: These changes don't look related to this PR.

File: package.json [hunk 2]
```diff
@@ -68,7 +67,6 @@
     "file-loader": "^0.8.5",
     "identity-obj-proxy": "^3.0.0",
     "imports-loader": "^0.6.5",
-    "in-publish": "^2.0.0",
```
Comment (Benaiah) [inline package.json]: These changes don't look related to this PR.

File: yarn.lock [hunk 3]
```diff
@@ -1,5 +1,7 @@
 # THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (Benaiah) [inline yarn.lock]: These changes don't look related to this PR.

File: src/components/UI/Sticky/Sticky.js [hunk 4]
```diff
@@ -81,7 +81,9 @@ export class StickyContext extends Component {
   }
 
   handleScroll = (event) => {
-    this.updateStickies(event.target);
+    if (event.target === this.ref) {
```
Comment (Benaiah) [inline src/components/UI/Sticky/Sticky.js]: LGTM

File: package.json (hunk 1 above)
Comment (erquhart) [inline package.json]: I actually did add this on purpose, I'll update the PR subject so that's clear.

**Comment by erquhart** [comment]
@Benaiah have another look, I removed prepublish on purpose.

**Comment by calavera** [comment]
LGTM
